tests/classes/local/groups_test.php

```diff
@@ -156,16 +161,19 @@ public function test_get_templates() {
             ->getMock();
 
         // Mock API response data.
-        $resdata = $this->mockapi->resdata('groups/search_success.json');
+        $resdata1 = $this->mockapi->resdata('groups/all_groups_page1.json');
+        $resdata2 = $this->mockapi->resdata('groups/all_groups_page2.json');
```

Comment:
I think it would be better to have separate JSON fixtures for `groups/search` because they are different endpoints even though their responses are identical.